Enterprise-CMCS/eAPD
# Tbolt/2738 Enhancements to session timeout alert to increase accessibility

Resolves #2738 

### Description
Updates the session timeout alert to only change on the minute vs. every second. Each change is displayed in the dialog and announced to the screen reader. At 1 minute the message reflects a slightly different message stating "less than 1 minute."

### Significant changes or possible side effects
Shouldn't impact anything outside of the alert dialog

### Automated test cases written

| Given | When | Then | Type (jest, tap, cypress) |
| ----- | ---- | ---- | ------------------------- |
| Logged in | idle for 10 minutes | alert shows message that 5 minutes remain before logout | jest |
| Logged in | idle for 14 minutes | alert shows message that less than 1 minute remains before logout | jest |

### Steps to manually verify this change

Note: A screen reader will need to be used to fully test this
1. Login and open an APD
2. Idle for 10+ minutes
3. Verify alert is shown. Verify message is read to screen reader at each minute mark
4. Verify logout is executed after the timeout. Verify message is read to screen reader

### This pull request is ready to code review when

      tests, refactor
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when


### This pull request is ready to review when the QA has


### This pull request can be merged when
